content/using/os/s3.md

```diff
@@ -12,15 +12,50 @@ To enable S3 support on your ship you'll need to first set up a bucket, and then
 
 ### Bucket Setup
 
-If you’re using AWS, it will have to support signature v2, not v4. We've found DigitalOcean's Spaces to work well also. The bucket has to be publicly readable, allow CORS from `*` origins, allow GET and PUT methods, and allow `*` headers. Your provider should have details on setting access permissions and CORS.
+We recommend DigitalOcean's Spaces, but any s3 provider that supports signature v2 should work. 
+
+The bucket has to be publicly readable, allow CORS from `*` origins, allow GET and PUT methods, and allow `*` headers. Specific instructions for DigitalOcean are below.
+
+In DigitalOcean you'll want to log into your existing account or create a new one if you don't already have one. You can use DigitalOcean for your S3 host even if your Urbit planet is hosted elsewhere.
+
+Click the green Create button at the top of the project page, then click Spaces.
+
+Make the following selections:
+- Select the datacenter region closest to you
+- Restrict File Listing (leave this as the default)
+- Choose a unique name (this is your bucket name)
+- Select a project (or leave the default if you don't have multiple)
+
+Once you've created your space you'll want to navigate to its settings in DigitalOcean.
+
+Then you'll want to set the CORS Configurations as specified above, this should look like the following image (your endpoint will vary depending upon region).
+![digitaloceans3](../../../public/images/digitaloceans3.png)
+![](../../../public/images/s3cors.png)
```

Comment:
need to upload to media.urbit.org s3

Reply:
I'm trying to get write access to this - do you have it? Are you able to give it to me?

Reply:
I have uploaded them. I have write access but not admin to give access:

`https://media.urbit.org/docs/user/digital-ocean-cors-settings.png`

`https://media.urbit.org/docs/user/digital-ocean-settings.png`